Source/Fuse.Reactive.Bindings/Instance.uno

```diff
@@ -330,7 +330,7 @@ namespace Fuse.Reactive
 			
 			This list should only be modified via the InsertNew, RemoveAt, and RemoveAll functions.
 		*/
-		List<WindowItem> _windowItems = new List<WindowItem>();
+		Fuse.Internal.ObjectList<WindowItem> _windowItems = new Fuse.Internal.ObjectList<WindowItem>();
```

Comment:
Maybe, I'm becoming reluctant to pull in namespaces for a single use lately as we have several name clashes across packages now, especially in the `Reactive.*` namespaces (where Instance is).